docs/releases/2.7.3b2.rst

```diff
@@ -18,11 +18,13 @@ Details of changes
 - :djadmin:`start` has been removed, use :djadmin:`runserver` instead.
 - :djadmin:`verify_user` and :djadmin:`purge_user` now accept multiple
   usernames.
-- :djadmin:`refresh_scores` now recalculates user scores and accept
+- :djadmin:`refresh_scores` now recalculates user scores and accepts
   multiple usernames.
 - :djadmin:`contributors` has two new options
   :option:`--since <contributors --since>` and
   :option:`--only-emails <contributors --only-emails>`.
+- :djadmin:`flush_cache` flushes `default`, `redis`, `stats` caches,
+  accepts `--rq`, `--stats`, `--default`, `--all` options.
```

Comment:
Some of the release notes changes made it into the typo fix commit for these lines